dash/_callback.py

```diff
@@ -464,23 +481,29 @@ def add_context(*args, **kwargs):
                 # Flatten grouping and validate grouping structure
                 flat_output_values = flatten_grouping(output_value, output)
 
-            _validate.validate_multi_return(
-                output_spec, flat_output_values, callback_id
-            )
-
             component_ids = collections.defaultdict(dict)
-            has_update = False
-            for val, spec in zip(flat_output_values, output_spec):
-                if isinstance(val, NoUpdate):
-                    continue
-                for vali, speci in (
-                    zip(val, spec) if isinstance(spec, list) else [[val, spec]]
-                ):
-                    if not isinstance(vali, NoUpdate):
-                        has_update = True
-                        id_str = stringify_id(speci["id"])
-                        prop = clean_property_name(speci["property"])
-                        component_ids[id_str][prop] = vali
+            if not no_output:
```

Comment:
`if not no_output:` that double negation 😅 is a bit hard on me.
Since it's a boolean maybe we could have something like `is....` or `should...`, but I am not sure what the output means in this context.